waterlog/src/index.scss

```diff
@@ -0,0 +1,8 @@
+body {
+    background-color: #253238;
+    text-align: center;
+    margin: 0;
+    padding: 0;
+    font-family: "Roboto";
```

Comment:
It's better to have some fall-back fonts setup here in case Roboto doesn't load for whatever reason to prevent the browser from defaulting to it's default font